nl_server/requirements.txt

```diff
@@ -1,16 +1,17 @@
 datasets==2.8.0
 diskcache==5.4.0
-Flask==1.1.4
+Flask==2.2.3
```

Comment:
I have tested that 2.0.0 (Flask and Werkzeug) would work for all the tests. Can you update that and this PR should be good to go

Reply:
done.